instructure/common-cartridge-viewer
# Quizzes answers

### Implements quizzes answers

_refs CM-437_
_fixes CM-555 CM-413_

Test plan
- Open a preview with assessments and confirm that its questions answers are shown
- add `hide-responses` to the URL and confirm that the answers are hidden
- Assessment should be called Quizz

Design: https://instructure.invisionapp.com/share/4CMYE0FUXTP#/screens/335558727_Commons-Quiz-01
